src/main/main.cpp

```diff
@@ -87,6 +87,7 @@ static const char USAGE_STRING[] =
     "  --jfrsync config  synchronize profiler with JFR recording\n"
     "  --libpath path    full path to libasyncProfiler.so in the container\n"
     "  --fdtransfer      use fdtransfer to serve perf requests\n"
+    "  --target-cpu      sample threads on a specific CPU (`-e cpu` only, default: -1)\n"
```

Comment:
Not `cpu` only; this can be used with any perf_event.

Reply:
a0934a229cbe60e472037c3e707fdc8b2cc6cb5d